panel/models/player.ts

```diff
@@ -320,8 +333,17 @@ export class PlayerView extends WidgetView {
     this.set_frame(this.model.end)
   }
 
+  updateSpeedButton(button: HTMLButtonElement, interval: number, originalSVG: string): void {
+    const fps = 1000 / interval
+    button.innerHTML = `${fps.toFixed(1)}<br>fps`
+    setTimeout(() => {
+      button.innerHTML = originalSVG
+    }, this.model.preview_duration) // Show for 1.5 seconds
```

Comment:
```suggestion
    }, this.model.preview_duration)
```

How does this behave if `preview_duration` is equal to zero.